src/main/java/spoon/support/comparator/CtLineElementComparator.java

```diff
@@ -24,7 +25,7 @@
  * Comparator of compile-time elements. Elements are sorted by position in
  * source files.
  */
-public class CtLineElementComparator implements Comparator<CtElement> {
+public class CtLineElementComparator implements Comparator<CtElement>, Serializable {
```

Comment:
This compactor is not efficient getColumn() and getLine() compute the column and the line based on the sourceStart.
It is way better to use getSourceStart() instead of getColumn()